agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/interfaces/LiteratureElasticSearchInterface.java

```diff
@@ -0,0 +1,62 @@
+package org.alliancegenome.indexer.indexers.curation.interfaces;
+
+import java.util.Map;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.DELETE;
+import jakarta.ws.rs.DefaultValue;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.rest.client.annotation.ClientHeaderParam;
+
+@ClientHeaderParam(name = "Cache-Control", value = "no-cache")
+@Path("/")
+@Produces(MediaType.APPLICATION_JSON)
+//@Consumes(MediaType.APPLICATION_JSON)
+public interface LiteratureElasticSearchInterface {
```

Comment:
This probally should be called ElasticSearchInterface as it can be used on any elastic search.

Reply:
@oblodgett not sure about this, this will connect to Blue team, probably do not want to mix this with calling to public index server ?